apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -3245,16 +3253,20 @@ static void tc_libc_stdio_tempnam(void)
 
 	unlink(ret1);
 	unlink(ret2);
+#if defined(CONFIG_FS_SMARTFS) && !defined(CONFIG_FS_AUTOMOUNT_TMPFS)
```

Comment:
Let's assume, both this conditional are true and user mounts tmpfs manually and runs fs tc. But at the end of the tc you are un mounting the mount point  which is not expected by user. Is it correct?

Reply:
yes, its getting unmounted. Will make sure that it will not get unmounted for this condition.